src/sidebar/styles.css

```diff
@@ -50,6 +50,10 @@ body .ck-editor .ck-editor__editable {
   padding-left: 20px;
 }
 
+.ck-editor__editable ol > li, .ck-editor__editable ul > li {
+  line-height: 1.5rem;
```

Comment:
Is 1rem colliding?

Reply:
@Natim it is. 1rem is 16px which I think matches the current P font-size, thus the reason for the overlapping.

Reply:
Ok thanks !